assets/base/i18n/en.json

```diff
@@ -520,7 +525,7 @@
   "threads.following": "Following",
   "threads.followMessage": "Follow Message",
   "threads.followThread": "Follow Thread",
-  "threads.newReplies": "{count} new {count, plural, one {reply} other {replies}}",
```

Comment:
`threads.newReplies` & `threads.replies` - We are using both. Added both in the Global Threads Screen PR.